src/runContextual.js

```diff
@@ -4,7 +4,7 @@ export default run => {
   if (isBrowser()) {
     // setTimeout is fixing this bug:
     // https://github.com/meteor/react-packages/issues/99
-    setImmediate(run);
+    setTimeout(run, 0);
```

Comment:
Hmm, I am not sure why that is but no biggie i guess, you can skip adding the `0` as [second arg since it is the default](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setTimeout)